src/kestrel/syntax/kestrel.lark

```diff
@@ -117,6 +117,14 @@ variables: VARIABLE ("," VARIABLE)*
 
 path_list: STIXPATH ("," STIXPATH)* -> valuelist
```

Comment:
Shall we remove `path_list` from the syntax?

Reply:
Done.